tests/MongoDB.Driver.Core.TestHelpers/XunitExtensions/RequirePlatform.cs

```diff
@@ -0,0 +1,99 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Linq;
+using Xunit;
+
+namespace MongoDB.Driver.TestHelpers
+{
+    public enum SupportedTargetFramework
+    {
+        Net452,
+        NetCoreApp11,
+        NetCoreApp21,
+        NetCoreApp30
+    }
+
+    public enum SupportedOperatingSystem
+    {
+        Windows,
+        Linux,
+        MacOS
+    }
+
+    public class RequirePlatform
+    {
+        #region static
+        public static RequirePlatform Create() => new RequirePlatform();
```

Comment:
The naming convention for the `RequireXyz` classes is `Check`:

    public static RequirePlatform Check() => new RequirePlatform();

Reply:
I think the method in RequiredServer has been named `Check` because the RequireServer method makes actually checking inside (on env variable). The method here doesn't make any checking, so I would leave the current naming unless you insist

Reply:
The method is called `Check` because we are "checking" some kind of requirement, regardless of how we do the check.

In this case we are "checking the platform requirements", so why not call the method `Check` like it is in all the other `RequireXyz` classes?